.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build LSOracle
+    container: ghcr.io/lnis-uofu/lsoracle-build:latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Build
+        run: |
+          mkdir build
+          cd build
```

Comment:
According to my experience, it seems that `cd` does not impact in Github Actions. That is why I was using multiple step to accomplish build process.

Reply:
It seems that build is successful. Never mind then.